app/javascript/controllers/transition_controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus'
+
+// Example:
+//
+// <div data-controller="transition" data-transition-toggle-class="!hidden">
+//   <div id="element-that-shows-and-hides" class="block" data-transition-target="me"></div>
+//   <a href="#" data-action="transition#toggleClass">click to toggle</a>
+// </div>
+//
+// Every element that is of target = me will have the class toggled.
+
+export default class extends Controller {
+  static classes = [ "toggle" ]
+  static targets = [ "me" ]
```

Comment:
Since this is the Transition controller, could this be "transitionable" or "transitioned"?

Reply:
Ah yes, good idea